### DISCUSSION TITLE
About Using the Mouse to Move Code in IDE Editing

### DISCUSSION THREAD

**Discussion by kongbaigit** [cpeditor/cpeditor#1184] [Ideas]
Many IDEs support selecting and moving code with a mouse. CP EDITOR can only move the code by copying and pasting, which is very inconvenient. It is recommended to add the function of selecting and moving the code directly with the mouse. Thank you very much for your efforts!

**Comment by ouuan** [comment]
Duplicate of #985

**Reply by kongbaigit** [reply]
But I think this is a feature that greatly affects the experience. After asking around, all the competition students in our school feel that this is not very good. Can you consider raising the priority?
